src/components/settings/engine-installation.tsx

```
@@ -0,0 +1,737 @@
+                  </div>
+                )}
+              </div>
+
+              <div
+                className={`p-4 border-2 rounded-lg cursor-pointer transition-colors ${
+                  method === 'binary'
+                    ? 'border-blue-500 bg-blue-50 dark:bg-blue-950/20'
+                    : 'border-gray-200 dark:border-gray-700 hover:border-gray-300 dark:hover:border-gray-600'
+                }`}
+                onClick={() => setMethod('binary')}
+              >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)